Conceptpower+Spring/src/integration-test/resources/output/conceptWithWordPosAndTypeId.xml

```diff
@@ -0,0 +1 @@
lHPS:wordnet_id></digitalHPS:conceptEntry><digitalHPS:conceptEntry><digitalHPS:id concept_id="WID-04502851-N-02-.22" concept_uri="http://www.digitalhps.org/concepts/WID-04502851-N-02-.22">http://www.digitalhps.org/concepts/WID-04502851-N-02-.22</digitalHPS:id><digitalHPS:lemma>.22</digitalHPS:lemma><digitalHPS:pos>noun</digitalHPS:pos><digitalHPS:description>a .22 caliber firearm (pistol or rifle)</digitalHPS:description><digitalHPS:conceptList>WordNet</digitalHPS:conceptList><digitalHPS:creator_id></digitalHPS:creator_id><digitalHPS:equal_to></digitalHPS:equal_to><digitalHPS:modified_by></digitalHPS:modified_by><digitalHPS:similar_to></digitalHPS:similar_to><digitalHPS:synonym_ids>WID-04502851-N-01-twenty-two,</digitalHPS:synonym_ids><digitalHPS:type ></digitalHPS:type><digitalHPS:deleted>false</digitalHPS:deleted><digitalHPS:wordnet_id>WID-04502851-N-02-.22</digitalHPS:wordnet_id></digitalHPS:conceptEntry><digitalHPS:conceptEntry><digitalHPS:id concept_id="CONf1b4b733-3994-458d-bea0-3fbc2110dc69" concept_uri="http://www.digitalhps.org/concepts/CONf1b4b733-3994-458d-bea0-3fbc2110dc69">http://www.digitalhps.org/concepts/CONf1b4b733-3994-458d-bea0-3fbc2110dc69</digitalHPS:id><digitalHPS:lemma>Almira Hart Lincoln Phelps</digitalHPS:lemma><digitalHPS:pos>noun</digitalHPS:pos><digitalHPS:description>American educator and author, particularly in the fields of botany, chemistry, and geology (1793-1884)</digitalHPS:description><digitalHPS:conceptList>VogonWeb Concepts</digitalHPS:conceptList><digitalHPS:creator_id>test</digitalHPS:creator_id><digitalHPS:equal_to></digitalHPS:equal_to><digitalHPS:modified_by>admin</digitalHPS:modified_by><digitalHPS:similar_to></digitalHPS:similar_to><digitalHPS:synonym_ids>[]</digitalHPS:synonym_ids><digitalHPS:type type_id="986a7cc9-c0c1-4720-b344-853f08c136ab" type_uri="http://www.digitalhps.org/types/TYPE_986a7cc9-c0c1-4720-b344-853f08c136ab">E21 Person</digitalHPS:type><digitalHPS:deleted>false</digitalHPS:deleted><digitalHPS:wordnet_id></digitalHPS:wordnet_id></digitalHPS:conceptEntry><digitalHPS:conceptEntry><digitalHPS:id concept_id="WID-00475142-N-02-1-hitter" concept_uri="http://www.digitalhps.org/concepts/WID-00475142-N-02-1-hitter">http://www.digitalhps.org/concepts/WID-00475142-N-02-1-hitter</digitalHPS:id><digitalHPS:lemma>1-hitter</digitalHPS:lemma><digitalHPS:pos>noun</digitalHPS:pos><digitalHPS:description>a game in which a pitcher allows the opposing team only one hit</digitalHPS:description><digitalHPS:conceptList>WordNet</digitalHPS:conceptList><digitalHPS:creator_id></digitalHPS:creator_id><digitalHPS:equal_to></digitalHPS:equal_to><digitalHPS:modified_by></digitalHPS:modified_by><digitalHPS:similar_to></digitalHPS:similar_to><digitalHPS:synonym_ids>WID-00475142-N-01-one-hitter,</digitalHPS:synonym_ids><digitalHPS:type ></digitalHPS:type><digitalHPS:deleted>false</digitalHPS:deleted><digitalHPS:wordnet_id>WID-00475142-N-02-1-hitter</digitalHPS:wordnet_id></digitalHPS:conceptEntry></conceptpowerReply>
```

Comment:
Type id search doesn't do a exact match search in master branch. This is fixed in develop branch.